modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/definition/details.jsp

```diff
@@ -154,7 +154,8 @@ if ((cpDefinition != null) && (cpDefinition.getExpirationDate() != null)) {
 					<liferay-asset:asset-categories-selector
 						className="<%= CPDefinition.class.getName() %>"
 						classPK="<%= cpDefinitionId %>"
-						groupIds="<%= new long[] {company.getGroupId()} %>"
+						groupIds="<%= new long[] {company.getGroupId(), cpDefinitionsDisplayContext.getSiteGroupId()} %>"
```

Comment:
@andrea-ale-sbarra I think the categories that we have to show are just the one connectected to global scope, not the one to the current site.  Why this was added?